code/datums/status_effects/neutral.dm

```diff
@@ -218,3 +218,34 @@
 /// Remove the image from the lwap user's screen
 /obj/effect/temp_visual/lwap_ping/proc/remove_mind(mob/living/looker)
 	looker.client?.images -= lwap_image
+
+/datum/status_effect/delayed
+	id = "delayed_status_effect"
+	status_type = STATUS_EFFECT_MULTIPLE
+	alert_type = null
+	var/prevent_signal = null
+	var/datum/callback/expire_proc = null
+
+/datum/status_effect/delayed/on_creation(mob/living/new_owner, new_duration, datum/callback/new_expire_proc, new_prevent_signal = null)
+	if(!duration || !new_expire_proc || !istype(new_expire_proc))
```

Comment:
Think you meant `new_duration` here. We don't want this being null. Also `!istype(new_expire_proc)` doubles as a null check, so no need to do `!new_expire_proc`.
```suggestion
	if(!new_duration || !istype(new_expire_proc))
```